vignettes/getting-started.Rmd

```diff
@@ -26,6 +27,7 @@ cleanup <- function() {
   if (fs::dir_exists("../inst/nonmem/2")) fs::dir_delete("../inst/nonmem/2")
   if (fs::dir_exists("../inst/nonmem/3")) fs::dir_delete("../inst/nonmem/3")
   if (fs::file_exists("../inst/nonmem/babylon.yaml")) fs::file_delete("../inst/nonmem/babylon.yaml")
+  if (fs::file_exists(file.path(nonmem_path, "1.yaml"))) fs::file_delete(file.path(nonmem_path, "1.yaml"))
```

Comment:
I'm not sure what the point is here? Doesn't this do the same thing as line 22? And if the point is the prefer this patten, then let's go ahead and change all of these lines.

Reply:
It does not do the same thing as line 22. If we comment this line out and run `check()`, we get the error

```
Error: System command 'R' failed, exit status: 1, stdout + stderr:
E> * checking for file ‘/data/home/seanw/git/rbabylon/DESCRIPTION’ ... OK
E> * preparing ‘rbabylon’:
E> * checking DESCRIPTION meta-information ... OK
E> * installing the package to build vignettes
E> * creating vignettes ... ERROR
E> --- re-building ‘getting-started.Rmd’ using rmarkdown
E> --- finished re-building ‘getting-started.Rmd’
E> 
E> --- re-building ‘parameter-labels.Rmd’ using rmarkdown
E> --- finished re-building ‘parameter-labels.Rmd’
E> 
E> --- re-building ‘using-based-on.Rmd’ using rmarkdown
E> Quitting from lines 66-70 (using-based-on.Rmd) 
E> Error: processing vignette 'using-based-on.Rmd' failed with diagnostics:
E> File already exists at /tmp/RtmpZci0UO/Rinst61a6bd9d387/rbabylon/nonmem/1.yaml. Either call `read_model()` to load model from YAML or use, `new_model(.overwrite = TRUE)` to overwrite the existing YAML.
E> --- failed re-building ‘using-based-on.Rmd’
```

The reason is a consequence of the `system.file()` change below: it creates `1.yaml` at a different location than the previous `new_model()` call. But, if we instead use the pattern `file.path(MODEL_DIR, "1")`, then line 22 will (continue to) work, and we can remove this line.